climetlab/core/ipython.py

```diff
@@ -42,15 +41,15 @@ def guess_which_ipython():
     if ipython_active.ipython_dir == "/deepnote-config/ipython":
         return ("deepnote", None)
 
-    if ipython_active.ipython_dir == "/home/jovyan/.ipython":
-        return ("jupyter-lab", None)
-
     if "google.colab" in repr(ipython_active.inspector):
         return ("colab", None)
 
     if "IPython.terminal" in repr(ipython_active.parent):
         return ("ipython", None)
 
+    if ipython_active.__class__.__name__== 'ZMQInteractiveShell':
```

Comment:
Good finding, I like using this very specific class name. And we can make it more general, latter on, if other environments require it.